tests/commentLikeDislike.js

```diff
@@ -0,0 +1,82 @@
+import chai from 'chai';
+import '@babel/polyfill';
+import chaiHttp from 'chai-http';
+import statusCode from '../src/helpers/constants/status.codes';
```

Comment:
Why don't we just use ``status`` instead of ``statusCode``? for me ``status`` has more meaning than this.